src/scratchtocatrobat/converter/converter.py

```diff
@@ -1423,3 +1429,34 @@ def _convert_answer_block(self):
                                                                                           self.script_context)
         return _variable_for(shared_global_answer_user_variable.getName())
 
+    @_register_handler(_block_name_to_handler_map, "createCloneOf")
+    def _convert_create_clone_of_block(self):
+        [base_sprite] = self.arguments
+        if isinstance(base_sprite, catformula.FormulaElement):
+            return catbricks.NoteBrick("Can't convert Clone-Brick with Formula as argument.\n")
+
+        if len(base_sprite) == 0:
+            return catbricks.NoteBrick("Can't convert Clone-Brick with no argument.\n")
+
+        if isinstance(base_sprite, basestring):
+            create_clone_of_brick = self.CatrobatClass(catbase.Sprite(base_sprite))
+            return create_clone_of_brick
+
+    @_register_handler(_block_name_to_handler_map, "timeAndDate")
+    def _convert_time_and_date_block(self):
+        [time_or_date] = self.arguments
+        switcher = {
+            "second": str(catformula.Sensors.TIME_SECOND),
+            "minute": str(catformula.Sensors.TIME_MINUTE),
+            "hour": str(catformula.Sensors.TIME_HOUR),
+            "day of week": str(catformula.Sensors.DATE_WEEKDAY),
```

Comment:
I saw it myself, but wasn't quite sure. 
I checked it (by trying it myself) and you can't change the starting day of the week system-wide.
You can only change it in the calendar, which doesn't effect the "day of week"-output of the App.
My emulator is also an American version of the Nexus and therefore the output is indeed of by one.

After thinking about this for a while, this workaround should be the correct solution. Because if Europeans write programs in Scratch they have to deal with this problem anyway. So programs would be incorrectly converted by us (without workaround).